spec/frise/loader_spec.rb

```diff
@@ -75,28 +62,153 @@ def validators.short_string(_, str)
       str.is_a?(String) && str.length < 5 || (raise "expected a short string, found #{str.inspect}")
     end
 
-    loader = Loader.new(
-      schema_load_paths: [fixture_path('_schemas')],
-      validators: validators
-    )
+    loader = Loader.new(validators: validators, exit_on_fail: true)
 
-    expect { loader.load(fixture_path('loader_test_1.yml'), true) }.to output(
+    expect { loader.load(fixture_path('loader_test1_schema.yml')) }.to output(
       "1 config error(s) found:\n" \
       " - At int: missing required value\n"
     ).to_stdout.and raise_error(SystemExit)
   end
 
   it 'should use an extra symbol table when provided' do
-    loader = Loader.new(
-      schema_load_paths: [fixture_path('_schemas')],
-      defaults_load_paths: [fixture_path('_defaults')]
+    loader = Loader.new(exit_on_fail: false)
+
+    conf = loader.load(fixture_path('loader_test2_all.yml'), '_id' => 'myobj')
```

Comment:
Can these extra symbols be hashes or arrays?

Reply:
They can, I added in 48a81d6 an object here to test that.